vello/src/scene.rs

```diff
@@ -80,8 +83,7 @@ impl Scene {
         if !self.encoding.encode_shape(clip, true) {
             // If the layer shape is invalid, encode a valid empty path. This suppresses
             // all drawing until the layer is popped.
-            self.encoding
-                .encode_shape(&Rect::new(0.0, 0.0, 0.0, 0.0), true);
+            self.encoding.encode_empty_shape();
         } else {
```

Comment:
Should this have a bit for the `bump_estimate` feature to count the 1 segment (empty) path or is that not needed since it is empty?